fedimint-client/src/oplog.rs

```diff
@@ -57,52 +84,73 @@ impl OperationLog {
         .await;
     }
 
+    #[deprecated(since = "0.6.0", note = "Use `paginate_operations_rev` instead")]
+    pub async fn list_operations(
+        &self,
+        limit: usize,
+        last_seen: Option<ChronologicalOperationLogKey>,
+    ) -> Vec<(ChronologicalOperationLogKey, OperationLogEntry)> {
+        self.paginate_operations_rev(limit, last_seen).await
+    }
+
     /// Returns the last `limit` operations. To fetch the next page, pass the
     /// last operation's [`ChronologicalOperationLogKey`] as `start_after`.
-    pub async fn list_operations(
+    pub async fn paginate_operations_rev(
         &self,
         limit: usize,
-        start_after: Option<ChronologicalOperationLogKey>,
+        last_seen: Option<ChronologicalOperationLogKey>,
     ) -> Vec<(ChronologicalOperationLogKey, OperationLogEntry)> {
+        const EPOCH_DURATION: Duration = Duration::from_secs(60 * 60 * 24 * 7);
+
+        let start_after_key = last_seen.unwrap_or_else(|| ChronologicalOperationLogKey {
+            creation_time: now(),
```

Comment:
Double check: timestamps created at `now()` are included. Just to not skip them in a racy, hard to debug way.

Reply:
Looks like we might because `Range` is not inclusive at the end? We could add a second, or maybe use `OperationId([0xff;32])`? Or maybe I'm confused.

My point is we should try to not theoretically skip anything that is already there if `last_seen == None`.

Reply:
You are right, since we deal with full precision time it's unlikely, but better safe than sorry, I'll add a commit to add some time.